src/main/java/Storage.java

```diff
@@ -0,0 +1,96 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+/**
+ * Represents the file used to store data.
+ */
+public class Storage {
+    private String filePath = "";
+    /**
+     * Default file path provides by user
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+    /**
+     * Loads the data from the storage file.
+     * @throws ErrorException if the default path is invalid
+     */
+    public ArrayList<Task> loadtxt() throws ErrorException {
+        ArrayList<Task> arrlist = new ArrayList<>();
+        try {
+            File txtfile = new File(filePath);
+            Scanner s = new Scanner(txtfile);
+            while (s.hasNext()) {
+            String txttasks = s.nextLine();
+            String[] txtsplit = txttasks.split(",");
+            if (txtsplit[0].equals("T")) {
+                String description = txtsplit[2];
+                Task todo = new Todo(description);
+                if (txtsplit[1].equals("1")) {
```

Comment:
The 3 inner `if txtsplit[1].equals("1") ...` and `arrList.add()` can probably be abstracted to out, since all 3 conditionals do the same thing.